test/e2e/kubectl.go

```diff
@@ -566,8 +566,15 @@ var _ = framework.KubeDescribe("Kubectl client", func() {
 			framework.SkipUnlessProviderIs("gke")
 			nsFlag := fmt.Sprintf("--namespace=%v", ns)
 			podJson := readTestFileOrDie(kubectlInPodFilename)
+
+			// Replace the host path to kubectl in json
+			podString := strings.Replace(string(podJson),
+				"$KUBECTL_PATH",
+				framework.TestContext.KubectlPath,
```

Comment:
@dims @saad-ali Does `framework.TestContext.KubectlPath` miss quotation mark here?
The string being replaced has quotation mark.